### PR TITLE
Added conversion

### PR DESCRIPTION
**Pull request by bohdan-titan**

### CODE REVIEW

File: public/main/client/handlers/no-core.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const storage = require("../storage");
 const auth = require("../auth");
 const wallet = require("../wallet");
 const { setProxyRouterConfig, getProxyRouterConfig } = require("../settings");
+const { getDefaultCurrencySetting, setDefaultCurrencySetting } = require("../settings");
```
Comment (lsheva) [inline public/main/client/handlers/no-core.js:10]: may be in one line

File: src/components/common/Root.js [hunk 2]
```diff
@@ -47,6 +47,16 @@ class Root extends React.Component {
...
       })
+      .then(() => {
+        this.props.client.getDefaultCurrencySetting().then(defaultCurr => {
+          const currency =
+            defaultCurr || this.props.sellerDefaultCurrency || 'BTC';
+          this.props.dispatch({
+            type: 'set-seller-currency',
+            payload: currency
+          });
+        });
```
Comment (lsheva) [inline src/components/common/Root.js:58]: please review this whole promise chain, the error handling appears to be missing

you should return the promise in then callback

File: src/components/contracts/modals/CreateContractModal.js [hunk 3]
```diff
@@ -20,12 +20,30 @@ import { lmrDecimals } from '../../../utils/coinValue';
...
+
+  const lmrEqualPrice = (profit, multiplier) =>
+    (multiplier * profit * lengthDays * speed * btcRate) / lmrRate;
+  console.log(
+    lmrEqualPrice(profit, 1),
+    lmrEqualPrice(profit, 1.05),
+    lmrEqualPrice(profit, 1.1),
+    lmrEqualPrice(profit, 1.15),
+    lmrEqualPrice(profit, 1.2)
+  );
```
Comment (lsheva) [inline src/components/contracts/modals/CreateContractModal.js:45]: cleanup

File: .github/workflows/auto-release-dev.yml [hunk 4]
```diff
@@ -86,6 +87,7 @@ jobs:
           #echo "ETH_NODE_ADDRESS_HTTP2=$ETH_NODE_ADDRESS_HTTP2" >> .env
           #echo "ETH_NODE_ADDRESS_HTTP3=$ETH_NODE_ADDRESS_HTTP3" >> .env
           echo "TITAN_LIGHTNING_POOL=$TITAN_LIGHTNING_POOL" >> .env
+          echo "DEFAULT_SELLER_CURRENCY=$DEFAULT_SELLER_CURRENCY" >> .env
```
Comment (alex-sandrk) [inline .github/workflows/auto-release-dev.yml:90]: do we really need this configurable from .env?

Comment (alex-sandrk) [reply .github/workflows/auto-release-dev.yml:90]: I just don't think we will use different values for 'dev', 'stg', 'prod'

File: src/components/common/Root.js (hunk 2 above)
Comment (bohdan-titan) [reply src/components/common/Root.js:58]: .Then() wrap callback in promise by itself, so it`s unnecessary to return promise if I don't want to send value to other callbacks.

**Review by alex-sandrk — APPROVED** [review]

**Review by lsheva — APPROVED** [review]

**Pull request merged**